zoomage_report.CURATED.tsv

```diff
@@ -55965,3 +55965,225 @@ cell type	normal human epidermal keratinocyte	http://purl.obolibrary.org/obo/CL_
+cell line	MOLT-3	http://www.ebi.ac.uk/efo/EFO_0005265	E-MTAB-9250	MOLT-3-C1	Anja Fullgrabe	13/01/21 18:40
+cell line	NALM-1	http://www.ebi.ac.uk/efo/EFO_0002244	E-MTAB-2770	run_NALM_1.1	Anja Fullgrabe	13/01/21 18:40
+cell line	RPMI-2650	http://www.ebi.ac.uk/efo/EFO_0006478	E-MTAB-3983	ERR983911	Anja Fullgrabe	13/01/21 18:40
+cell line	SBC-3	ttp://www.ebi.ac.uk/efo/EFO_0002854	E-MTAB-3983	ERR905179	Anja Fullgrabe	13/01/21 18:40
+cell type	CD8-positive T-Lymphocytes	http://www.ebi.ac.uk/efo/EFO_0004219	E-GEOD-50119	GSM1214439	Anja Fullgrabe	13/01/21 18:40
+cell type	CD8-positive T-Lymphocytes	http://www.ebi.ac.uk/efo/EFO_0004219	E-GEOD-50121	GSM1214463	Anja Fullgrabe	13/01/21 18:40
+cell type	CD8-Positive T-Lymphocytes	http://www.ebi.ac.uk/efo/EFO_0004219	E-MEXP-3635	HCV-2	Anja Fullgrabe	13/01/21 18:40
+cell type	CD8-Positive T-Lymphocytes	http://www.ebi.ac.uk/efo/EFO_0004219	E-MTAB-6091	G33_PS02B	Anja Fullgrabe	13/01/21 18:40
+cell type	neuroblast (sensu Vertebrata)	http://purl.obolibrary.org/obo/CL_0000031	E-MTAB-9315	ERR4319000	Anja Fullgrabe	13/01/21 18:40
+cell type	neuroblast (sensu Vertebrata)	http://purl.obolibrary.org/obo/CL_0000031	E-MTAB-9338	ERR4336607	Anja Fullgrabe	13/01/21 18:40
+cell type	neuroblast (sensu Vertebrata)	http://purl.obolibrary.org/obo/CL_0000031	E-MTAB-9340	ERR4336744	Anja Fullgrabe	13/01/21 18:40
+cell type	pneumocyte	http://purl.obolibrary.org/obo/CL_0000322	E-GEOD-147507	SRR11412281	Anja Fullgrabe	13/01/21 18:40
+cell type	human mammary epithelial cell (HMEC)	http://www.ebi.ac.uk/efo/EFO_0001188	E-GEOD-26284	SRR315319	Anja Fullgrabe	13/01/21 18:40
+cell type	human mesenchymal stem cell from adipose tissue (hMSC-AT)	http://www.ebi.ac.uk/efo/EFO_0005023	E-GEOD-26284	SRR534334	Anja Fullgrabe	13/01/21 18:40
+cell type	human pericytes from placenta (hPC-PL)	http://www.ebi.ac.uk/efo/EFO_0005024	E-GEOD-26284	SRR534317	Anja Fullgrabe	13/01/21 18:40
+cell type	human skeletal muscle myoblast (HSMM)	http://purl.obolibrary.org/obo/CL_0000515	E-GEOD-26284	SRR307932	Anja Fullgrabe	13/01/21 18:40
+cell type	human umblical vein endothelial cells (HUVEC)	http://www.ebi.ac.uk/efo/EFO_0002451	E-GEOD-26284	SRR307906	Anja Fullgrabe	13/01/21 18:40
+cell type	intestinal epithelial cells (IECs)	http://purl.obolibrary.org/obo/CL_0002563	E-GEOD-55356	GSM1334671	Anja Fullgrabe	13/01/21 18:40
+cell type	CD4 T cell	http://purl.obolibrary.org/obo/CL_0000624	E-GEOD-14926	GSM372800	Anja Fullgrabe	13/01/21 18:40
+cell type	CD8 T cell	http://purl.obolibrary.org/obo/CL_0000625	E-GEOD-14926	GSM372809	Anja Fullgrabe	13/01/21 18:40
+cell type	CD8 T cell	http://purl.obolibrary.org/obo/CL_0000625	E-GEOD-53358	GSM1290025	Anja Fullgrabe	13/01/21 18:40
+cell type	CD8 T cell	http://purl.obolibrary.org/obo/CL_0000625	E-GEOD-58078	GSM1400450	Anja Fullgrabe	13/01/21 18:40
+cell type	CD8 T cell	http://purl.obolibrary.org/obo/CL_0000625	E-MTAB-7462	ERR3458886	Anja Fullgrabe	13/01/21 18:40
```

Comment:
@anjaf I might be being a bit mean here, but why not map this to the previous EFO term you used for CD8 positive T lymphocyte? https://www.ebi.ac.uk/ols/ontologies/efo/terms?short_form=EFO_0004219

Reply:
I never liked this term "CD8-Positive T-Lymphocytes" (http://www.ebi.ac.uk/efo/EFO_0004219) much as it is sort of out of place in the ontology hierarchy (under regulatory T cell), has no equivalent "CD4-postive T-lymphocyte" and is not following EFO style (plural, upper case). 
I'm not an expert on lymphocytes. Is there anything that speaks against matching to CL_0000625?

Reply:
You're right, this was inconsistent with the older term mappings. "CD8 T cells" and "CD8-positive T-Lymphocyte" are now both mapped to http://purl.obolibrary.org/obo/CL_0000625.